CrosswordMe.py

```diff
@@ -1,17 +1,46 @@
+from random import randint
+
 #input
 
-words = []
+words = ["one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"]
 user_input = ""
+print(len(words))
 
-while 1 == 1:
+"""while 1 == 1:
     user_input = input("Word: ")
     if user_input.lower() == "done":
         break
-    words.append(user_input)
+    words.append(user_input)"""
 
 # creating the puzzle
+## horizontals
+line = randint(0,9) #First line to write in
+lines_used = [] #Empty array which contains lines which have been written in already (can probably be removed when its possible to write multiple words in one line)
+
+puzzle = [["$" for x in range(10)] for y in range(10)] #Creates 10x10 array filled with "$"
```

Comment:
don't hardcode stuff like sizes. Rather make a variable `line_count` and do:
```puzzle = [["$" for x in range(line_count)] for y in range(line_count)]```
that way you only have to change the size in one place should that be encessary.
And later on, you could even pass the size as an argument or calculate the size from the number of words you get as user input.